packages/swapper/src/swappers/LifiSwapper/getTradeRate/getTradeRate.ts

```diff
@@ -11,9 +11,10 @@ export const getTradeRate = async (
   deps: SwapperDeps,
   lifiChainMap: Map<ChainId, ChainKey>,
 ): Promise<Result<LifiTradeRate[], SwapErrorRight>> => {
-  const rate = (await getTrade(input, deps, lifiChainMap)) as Result<
-    LifiTradeRate[],
-    SwapErrorRight
-  >
+  const rate = (await getTrade({
+    input,
+    deps,
+    lifiChainMap,
+  })) as unknown as Result<LifiTradeRate[], SwapErrorRight>
```

Comment:
Unless my LSP is broken, this change doesnt seem to be required

Reply:
tsc def. sad without!

<img width="1438" alt="image" src="https://github.com/user-attachments/assets/82348cfc-d2c5-4875-9f7b-940fe99266d6">

Reply:
We do need the cast, but I don't think we need to go via `unknown`:

```suggestion
  })) as Result<LifiTradeRate[], SwapErrorRight>
```

Reply:
Nice catch! Removed the `as unknown` cast in e52c6f7d6b and followed up with 214784c534